src/client.js

```diff
@@ -7,7 +7,7 @@ import ServiceRegistry from './services';
 import * as common from './common';
 import ExtJSON from 'mongodb-extjson';
 import queryString from 'query-string';
-import { deprecate, } from './util';
+import { deprecate } from './util';
```

Comment:
this is sort of unrelated to the purpose of this ticket but - if we're planning to do a breaking (major) version for this change, maybe this is the right time to just remove the deprecated function bindings in this class. thoughts? cc:  @edaniels

Reply:
We'll need to check all our docs/samples and make sure that any places where those deprecated functions are used get updated to the new stuff.